marconi-chain-index/src/Marconi/ChainIndex/Indexers/Utxo.hs

```diff
@@ -186,7 +183,7 @@ data Utxo = Utxo
 $(makeLenses ''Utxo)
 
 instance Ord Utxo where
-  compare u u' = compare (u ^. address, u ^. txIn) (u' ^. address, u' ^. txIn)
+  compare u1 u2 = compare (u1 ^. txIn) (u2 ^. txIn)
```

Comment:
not comparing the address?

Reply:
I think we've discussed it and conclude that the `txIn` would be enough to differentiate Utxos. I honestly like to drop this ord instance and to use `compareOn` when needed, and use `Set` instead of `Map` when needed, but it can be done in another story.

Reply:
I think we've discussed it and conclude that the `txIn` would be enough to differentiate Utxos. I honestly like to drop this ord instance and to use `compareOn` when needed, and use `Set` instead of `Map` when needed, but it can be done in another story.